src/components/list/SearchList.tsx

```diff
@@ -32,6 +33,16 @@ const SearchList = forwardRef(
     const a11yIndex = (Math.floor(items.length / blockSize) - 1) * blockSize;
     const loadedMoreAmount = hasNext ? blockSize : count - items.length;
 
+    if (count === 0) {
+      return (
+        <InfoTemplate
+          icon={IconSearch}
+          title="Sorry no search results were found for the search criteria you selected"
+          description="Try changing your search criteria and you're sure to find something to do!"
+        />
```

Comment:
Minor nitpicky comment: All the other placeholder texts in the application are in finnish, so this being only one in english causes it to stick out.

Reply:
The design had an English translation so that's why--I didn't want to go about and translate it myself as there was talk that we should avoid our own translations and leave that work to translators or content producers.

I agree with you though. My thinking was that as I was going to implement translations next, this inconsistency would not be in the application for long.